### PR TITLE
[android] Various fixes for android

### PR DESCRIPTION
**Pull request by todi1856**
- [android] Apply process for touch positions, on Android y value is inverted
- Added InputRuntime.screenSize
- [android] Added input controls, comments, tests for sensors
- Rewrote sensor tests in CoreTest.cs

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/Devices/Touchscreen.cs [hunk 1]
```diff
@@ -62,6 +62,21 @@ public FourCC GetFormat()
...
+        {
+#if UNITY_EDITOR
+            return value;
+#elif PLATFORM_ANDROID
+            return new Vector2(value.x, InputRuntime.s_Instance.screenSize.y - value.y);
+#else
+            return value;
+#endif
+        }
+    }
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/InputSystem/Devices/Touchscreen.cs:77]: This one I don't like so much. If there's no need for a processor, there simply shouldn't be a processor on the control. For now, let's take it and then refine it.

**Review by Rene-Damm — APPROVED** [review]

File: Packages/com.unity.inputsystem/InputSystem/IInputRuntime.cs [hunk 2]
```diff
@@ -112,6 +112,7 @@ public unsafe interface IInputRuntime
         InputUpdateType updateMask { set; }
 
         ScreenOrientation screenOrientation { get; }
+        Vector2 screenSize { get; }
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/InputSystem/IInputRuntime.cs:115]: Looking at these two properties I find myself hesitant. They proliferate what I think is simply a stupid assumption in the old Unity APIs -- that there is a single screen on the system. We've hardcoded and spread that into several places and then awkwardly worked around it with the "multi display" APIs.

Not sure how to best handle this but would prefer a solution that doesn't involve adding more APIs cementing what I think is a design fallacy.

**Comment by todi1856** [comment]
@Rene-Damm you're right, at least for screenSize which is per screen, and we can have multiple screens...

as for screenOrientation, there cannot be multiple instances of it.

**Comment by todi1856** [comment]
So looking ahead, we'll probably need screen size and window size, and probably screen offset in some case, as on some platforms (for ex., UWP), you can have unity screen occupy 1/3 of screen with specific offset.

**Comment by todi1856** [comment]
Maybe it's worth to add ////FIXME: at those lines with comments

**Pull request merged**